pkg/azdiskschedulerextender/azdiskschedulerextender.go

```diff
@@ -63,16 +67,17 @@ type azDriverNodesMeta struct {
 }
 
 type azVolumeAttachmentsMeta struct {
-	volumes []*v1alpha1Meta.AzVolumeAttachment
-	err     error
+	volumeAttachments []*v1alpha1Meta.AzVolumeAttachment
+	err               error
 }
 
 func init() {
 	flag.StringVar(&criNamespace, "driver-object-namespace", consts.DefaultAzureDiskCrdNamespace, "The namespace where driver related custom resources are created.")
 }
 
 func initSchedulerExtender(ctx context.Context) {
-	pvcToPvMapCache = cachedMapping{memo: make(map[string]*cacheEntry)}
+	pvcToPvMapCache = sync.Map{}
+	pvToVolumeMapCache = sync.Map{}
```

Comment:
`pvToDiskNameMapCache`? Volume name is used is used in so many places (extender args, pod, pv, pvc details) that it is redundant and makes the purpose of this cache ambiguous. We should pick a name that more clearly states what this mapping is for.